include/deal.II/base/array_view.h

```diff
@@ -524,6 +524,46 @@ make_array_view (ElementType *const begin, ElementType *const end)
 
 
 
+/**
+ * Create a view from an ArrayView itself.
+ *
+ * This function is used for @p const references to objects of ArrayView type.
+ * It only exists for compatibility purposes.
+ *
+ * @param[in] array_view The ArrayView that we wish to make a copy of.
+ *
+ * @relates ArrayView
+ */
+template <typename Number>
+inline
+ArrayView<const Number>
+make_array_view (const ArrayView<Number> &array_view)
```

Comment:
@masterleinad I think that you added some of the other `make_array_view` functions previously. Are you happy with these two additions (they help instantiation of some functions)?

Reply:
Sure. In the end, I prefer to have external `make_array_functions` instead of modifying the class `ArrayView` itself. If this helps with compatibility, I am totally fine with it.

Reply:
Great, thanks for the confirmation!